src/main/resources/templates/project-commit-graph.ftl

```diff
@@ -0,0 +1,129 @@
+[#macro listTags repository commitId]
+    [#list repository.getTags() as tag]
+        [#if tag.getCommit().getCommit() == commitId]
+        <span class="label label-primary">${tag.getSimpleName()}</span>
+        [/#if]
+    [/#list]
+[/#macro]
+
+[#import "macros.ftl" as macros]
+[#import "components/project-frameset.ftl" as projectFrameset]
+[#import "components/commit-row.ftl" as commitRow]
+
+[@macros.renderHeader i18n.translate("section.projects") /]
+[@macros.renderMenu i18n user /]
+<div class="container">
+
+[#if repositoryEntity?? && repositoryEntity?has_content]
+    [@projectFrameset.renderBreadcrumb i18n group![] repositoryEntity/]
+[/#if]
+
+    <div class="row">
+        <div class="col-md-10 col-md-offset-2">
+            <h4 style="line-height:34px; margin-top:0;">Nice graphs &nbsp; 😍</h4>
```

Comment:
Can you please replace all "Nice graphs" occurrences with a property within our localisation file? (so `${i18n.translate("header.graphs")}` and then in `devhub_en.properties` you add: `header.graphs = ...`. We may reconsider to change the title too. I would keep the emoji.